lms/djangoapps/teams/views.py

```diff
@@ -0,0 +1,449 @@
+
+            * language: Optionally specifies which language the team is associated with.
+
+            * membership: A list of the users that are members of the team. See membership endpoint for more detail.
+
+            For all text fields, clients rendering the values should take care
+            to HTML escape them to avoid script injections, as the data is
+            stored exactly as specified. The intention is that plain text is
+            supported, not HTML.
+
+            If the user is not logged in, a 403 error is returned.
+
+            If the specified team does not exist, a 404 error is returned.
+
+        **Response Values for PATCH**
+
+            Only staff can patch teams. If the specified team does not exist, a
+            404 error is returned.
+
+            If "application/merge-patch+json" is not the specified content type,
+            a 415 error is returned.
+
+            If the update could not be completed due to validation errors, this
+            method returns a 400 error with all error messages in the
+            "field_errors" field of the returned JSON.
+    """
+
+    authentication_classes = (SessionAuthentication, OAuth2Authentication)
+    permission_classes = (permissions.IsAuthenticated, IsStaffOrReadOnly)
+    lookup_field = 'team_id'
+    serializer_class = CourseTeamSerializer
+    parser_classes = (MergePatchParser,)
+
+    def get_queryset(self):
+        """
+        Returns the queryset used to access the given team.
+        """
+        return CourseTeam.objects.all()
+
+
+class TopicListView(GenericAPIView):
+    """
+    **Use Cases**
+
+    Retrieve a list of topics associated with a single course.
+
+    **Example Requests**
+
+    GET /api/team/v0/topics/?course_id={course_id}
+
+    **Response Values for GET**
+
+    The following options can be specified as query parameters:
+
+    * course_id: Filters the result to topics belonging to the given course (required).
+
+    * order_by: Orders the results. Currently only 'name' is supported, and is also the default value.
+
+    * page_size: Number of results to return per page.
+
+    * page: Page number to retrieve.
+
+    If the course_id is not given or an unsupported value is passed for order_by, returns a 400 error.
+
+    If the user is not logged in or not enrolled in the course, returns a 403 error.
+
+    If the course does not exist, returns a 404 error.
+
+    Otherwise, a 200 response is returned containing the following fields:
+
+    * count: The total number of topics matching the request.
+
+    * next: The URL to the next page of results, or null if this is the last page.
+
+    * previous: The URL to the previous page of results, or null if this is the first page.
+
+    * num_pages: The total number of pages in the result.
+
+    * results: A list of the topics matching the request.
+
+        * id: The topic's unique identifier.
+
+        * name: The name of the topic.
```

Comment:
The User API uses just ``name`` for accounts, so that would be my preference here. IMO we only need ``display_name`` when there's already a low-level ``name`` field.